### PR TITLE
Correct pg_user name to oc_nextcloud

### PR DESCRIPTION
**Pull request by robeatoz**
I recently transitioned my Nextcloud instance to Nextcloud AiO, following the guidelines outlined in [migration.md](https://github.com/nextcloud/all-in-one/blob/main/migration.md). 
These instructions recommend using the `pg_user` `ncadmin`.

To verify this, I queried all `pg_user` entries in the Postgres database created by the Nextcloud AiO Master container using the following command:

```bash
docker exec -it nextcloud-aio-database psql -U oc_nextcloud nextcloud_database -c "SELECT usename FROM pg_user;"
```

This query yielded the following results:

```text
   usename
--------------
 nextcloud
 oc_nextcloud
(2 rows)
```

Based on these findings, I modified the migration instructions to utilize the `pg_user` `oc_nextcloud` instead of `ncadmin`. 
With this adjustment, I successfully finished my migration process.

I did not try to use the `ncadmin` user.

### CODE REVIEW

**Comment by szaimen** [comment]
Hi, is this maybe already fixing your problem? https://github.com/nextcloud/all-in-one/pull/4279

**Comment by robeatoz** [comment]
I made a mistake during the merge process, and I apologize for any confusion caused. The conflicts have now been resolved.

I'm puzzled by how the migration mentioned in https://github.com/nextcloud/all-in-one/pull/4279 was successful using the PostgreSQL user `ncadmin`. As noted earlier, my database only contains the following PostgreSQL users:

```bash
   usename
--------------
 nextcloud
 oc_nextcloud
(2 rows)
````

Should there be an issue with my setup, I am prepared to retract this merge request. I appreciate your guidance!

**Comment by FaySmash** [comment]
> I made a mistake during the merge process, and I apologize for any confusion caused. The conflicts have now been resolved.
> 
> I'm puzzled by how the migration mentioned in #4279 was successful using the PostgreSQL user `ncadmin`. As noted earlier, my database only contains the following PostgreSQL users:
> 
> ```shell
>    usename
> --------------
>  nextcloud
>  oc_nextcloud
> (2 rows)
> ```
> 
> Should there be an issue with my setup, I am prepared to retract this merge request. I appreciate your guidance!

So as far as I understood it the process is as follows:

- export the dump (which has nextcloud as the owner)
- replace all occurrences of the owner `nextcloud` with `ncadmin`
-  follow the instructions and place the dump into the PostgreSQL container import directory
- at the first boot the, PostgreSQL container imports the dump **but after the successful import sets the nextcloud user as table owner and taking away all permissions from the ncadmin user**
- after the import has finished, the dump gets deleted out of the import directory and the user ncadmin is left but with no permissions (at least that was the case for me)
- the ncadmin user can safely be deleted

TL;DR: the ncadmin user seems to be just a placeholder for the automated import process after which the real owner is set to nextcloud and ncadmin can be deleted

**Comment by robeatoz** [comment]
Thanks, @FaySmash, for the clarification! It's intriguing that `ncadmin` appears only in [migration.md](https://github.com/nextcloud/all-in-one/blob/main/migration.md).

I propose renaming the PostgreSQL user as suggested in the merge request for simplicity. Alternatively, we could clarify in the documentation that `ncadmin` is a temporary PostgreSQL user.

Ultimately, the decision rests with the maintainer.

Appreciate all the work on this valuable project!

**Comment by szaimen** [comment]
> Alternatively, we could clarify in the documentation that `ncadmin` is a temporary PostgreSQL user.

Sounds good. Would you adjust the PR? :)

**Comment by robeatoz** [comment]
I've updated the merge request. Should you identify a more suitable location or wording for the note I added, please feel free to make adjustments. Thank you for your support!

File: migration.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The procedure for migrating the files and the database works like this:
         ```
     1. Create a new database by running:
         ```
-        export PG_USER="oc_nextcloud"
+        export PG_USER="ncadmin"
```
Comment (szaimen) [inline migration.md]: @robeatoz maybe simply do it like this?
```suggestion
        export PG_USER="ncadmin" # This is a temporary user that gets created for the dump but is then overwritten by the correct one later on
```

**Comment by szaimen** [comment]
Thanks @robeatoz , maybe it would make sense to create a new PR with the change that I suggested?

**Comment by robeatoz** [comment]
I created https://github.com/nextcloud/all-in-one/pull/4334 for a clean Merge-Request. Thank you very much :)
